lib/service_check.pm

```diff
@@ -139,14 +154,21 @@ sub install_services {
 }
 
 sub check_services {
-    my $service = shift;
+    my ($service) = @_;
     foreach my $s (keys %$service) {
         my $srv_proc_name = $service->{$s}->{srv_proc_name};
         my $support_ver   = $service->{$s}->{support_ver};
         if (grep { $_ eq $hdd_base_version } split(',', $support_ver)) {
+            # service check after migration. if the we've set up service check
+            # function, we don't need following actions to check the service.
+            if (exists $service->{$s}->{service_check_func}) {
+                $service->{$s}->{service_check_func}->('after');
```

Comment:
do we even need to give the parameter 'after'? In the function full_apache_check you are only checking `if ($stage eq 'before')`